pom.xml

```diff
@@ -2082,18 +2082,47 @@
         <plugin>
           <groupId>com.hubspot.maven.plugins</groupId>
           <artifactId>prettier-maven-plugin</artifactId>
-          <version>0.22</version>
-          <configuration>
+          <version>1.0-deprecate-SNAPSHOT</version>
```

Comment:
we can't reference SNAPSHOT versions in the basepom; will need to cut a release of the plugin from this branch.

Reply:
We can also just not use this branch